tests/test_pipeline.py

```diff
@@ -195,7 +195,7 @@ def test_label_006_website_inspection(self):
         inspection = analyze(label_storage, self.ocr, self.gpt)
 
         # Assertions for website fields
-        self.assertEqual(inspection.company_website, "www.activeagriscience.com")
+        self.assertIn("activeagriscience.com", str(inspection.organizations))
```

Comment:
This was done as per requirements for #60. Please try with a prompt similar to the old one. If not working, we'll do it in the validation function.